sdk/dotnet/Awsx/Inputs/RoleWithPolicyArgs.cs

```diff
@@ -78,14 +78,14 @@ public InputList<string> ManagedPolicyArns
         public Input<string>? PermissionsBoundary { get; set; }
 
         [Input("policyArns")]
-        private List<string>? _policyArns;
+        private InputList<string>? _policyArns;
 
         /// <summary>
         /// ARNs of the policies to attach to the created role.
         /// </summary>
-        public List<string> PolicyArns
+        public InputList<string> PolicyArns
```

Comment:
`public sealed class InputList<T> : Input<ImmutableArray<T>>, IEnumerable, IAsyncEnumerable<Input<T>>`

This is breaking I think.

Reply:
I *think* dotnet has an implict conversion from `List<T>` to `InputList<T>`

Reply:
I will try it out just to be sure. I was also under the impression, but better safe than sorry.

Reply:
Oh that's true, I forgot about implicit conversions. That may be nice.